org.lflang.tests/src/org/lflang/tests/cli/CliToolTestFixture.java

```diff
@@ -87,6 +92,36 @@ public ExecutionResult run(Path wd, String... args) {
         return new ExecutionResult(out, err, exitCode);
     }
 
+    /**
+     * Run and return and Lfc instance with the given arguments, in the given
+     * working directory.
+     *
+     * @param wd working directory
+     * @param args Arguments
+     * @return The execution result
+     */
+    public LfcObjExecutionResult runLfcObj(Path wd, String... args) {
```

Comment:
I don't think we need to add that much code for this. We could just subclass `CliToolTestFixture` and save the `Lfc` instance in there. Otherwise there is a lot of code duplication.